src/SearchParameters/index.js

```diff
@@ -443,6 +454,22 @@ SearchParameters.prototype = {
     } );
   },
   /**
+   * addTagRefinement adds a tag to the list used to filter the results
+   * @param {string} tag tag to be added
+   * @return {SearchParameters}
+   */
+  addTagRefinement : function addTagRefinement( tag ) {
+    if( this.tagFilters ) {
+      return this.setQueryParameter( "tagFilters", undefined )
```

Comment:
If it's on purpose he could do `helper.clearTagRefinements().addTagRefinement("foo").search()` or something like that. We need to ensure he can do it (with `clearTagRefinements()` or `setQueryParameters('tagFilters', undefined)` but I would not do it automagically.